samples/matter/template/sample.yaml

```diff
@@ -76,3 +76,8 @@ tests:
       - nrf54h20dk/nrf54h20/cpuapp
     platform_allow: nrf54h20dk/nrf54h20/cpuapp
     tags: sysbuild
+  samples.matter.template.nrf54h20.suit:
```

Comment:
I would suggest sample.matter.template.smp_dfu with the new board instead of creating new configuration.

Reply:
ok